src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -13600,6 +13600,61 @@ public sealed partial class SuppressGCTransitionAttribute : System.Attribute
     {
         public SuppressGCTransitionAttribute() { }
     }
+    public enum UnmanagedType
```

Comment:
CC. @AaronRobinsonMSFT, @jkoritzinsky, @ericstj on this part.

This PR requires the enum to be moved down to `System.Runtime`. This is because the `unmanaged` constraint requires this type to be resolvable and the xplat `Load<T>(T* source)` + `Store<T>(Vector128<T> source, T* destination)` methods require the unmanaged constraint.

We technically could "explode" the surface area by making it non-generic if there is a potential issue here, but my expectation is there won't be.

Reply:
I'm fine moving the enum down so we can use the `unmanaged` constraint in CoreLib.

(For reference for others who read this, Roslyn emits a `modreq(System.Runtime.InteropServices.UnmanagedType)` as part of the signature of a generic type parameter constrained to `T : unmanaged`)

Reply:
>I'm fine moving the enum down so we can use the unmanaged constraint in CoreLib.

+1

/cc @dotnet/interop-contrib

Reply:
+1 Moving down should be fine.  Since this is inside the shared framework I don't expect fallout, we've done similar moves already this release (crypto).